README.md

```diff
@@ -50,7 +50,9 @@ You can run this repo virtually by using GitHub Codespaces or VS Code Remote Con
 #### Project Initialization
 
 1. Create a new folder and switch to it in the terminal
-1. Run `azd login`
+1. Run `azd login` 
+   1. If you're in multi-tenant organization, please use `azd login --tenant-id {YOUR_TENANT_ID}`
+   1. If you're using dev-container, please use `azd login --use-device-code`
```

Comment:
```suggestion
   1. If you're using VS Code DevContainer or GitHub Codespaces, please use `azd login --use-device-code`
```

Reply:
I add `azd login --tenant-id {YOUR_TENANT_ID} --use-device-code` initially, will combine with this.

Reply:
Refined. @jongio , @vhvb1989